docs/source/export.rst

```diff
@@ -0,0 +1,113 @@
+Exporting NWB files
+===================
+
+Export is a new feature in PyNWB 2.0 (adapted from HDMF 2.0). You can use export to take an
+:py:class:`~pynwb.file.NWBFile` that was read from a file and write it to
+a different file, with or without modifications to the `NWBFile` in memory.
+The in-memory `NWBFile` being exported will be written to the exported file as if it was never read from a file.
+
+To export an `NWBFile`, first read the `NWBFile` from a file, then create a new
+:py:class:`~pynwb.NWBHDF5IO` object for exporting the data, then call
+:py:meth:`~pynwb.NWBHDF5IO.export` on the
+:py:class:`~pynwb.NWBHDF5IO` object, passing in the IO object used to read the `NWBFile`
+and optionally, the `NWBFile` itself, which may be modified in memory between reading and exporting.
+
+For example:
+
+.. code-block:: python
+
+   with NWBHDF5IO(self.read_path, mode='r') as read_io:
+       with NWBHDF5IO(self.export_path, mode='w') as export_io:
+           export_io.export(src_io=read_io)
+
+FAQ
+---
+
+Can I read an `NWBFile` from disk, modify it, and then export the modified `NWBFile`?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I would consider rephrasing the questions from "Can I.." and "What if.." questions more to "How to..." questions, e.g., 

- "How to create a modified version of an existing NWBFile, while preserving the original file?"
- "How to export a newly instantiated `NWBFile`?
- The question "What happens to object IDs when I export?" could possible be split into two parts. Part 1 "How can I preserve object IDs when exporting a file? and Part 2, "How can I update the object Ids in an NWBFile for export?"

I think the changes needed to the document itself should be fairly minimal, i.e,. I think the content of the document addresses the right problems. However, I think a "How to..." question style may be more approachable to users, as it tells a story of how to address a real-world problem a user may face, whereas the FAQ style may appear less approachable as it gives the appearance to focus more on technical details of the implementation.

Reply:
I restructured this page. Some of the FAQ has now been moved to the docstring of `export`. Some was already included as part of the Add/remove Containers tutorial. The rest, I put into "How do I" format